music/demos/midi_me.html

```diff
@@ -0,0 +1,77 @@
+
+<!-- Copyright 2017 Google Inc. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================-->
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+  <link rel="stylesheet" href="./style.css">
+  <title>MidiME</title>
+  <style>
+    input[type="file"] {
+      width: 0;
+      height: 0;
+      opacity: 0;
+      cursor: pointer;
+      display: none;
+    }
+  </style>
+</head>
+<body>
+  <h1>MidiME</h1>
+  <p><code>MidiME</code> is a variational autoencoder that lets you personalize
+    your own <code>MusicVAE</code> model with just a little data, so that samples
+    from <code>MidiME</code> sound closer to your provided data.</p>
+  <h2>Input data</h2>
+  <section>
+    <p>This is what you want the samples to sound like. Note that MusicVAE is a
+      monophonic model, so if you use a polyphonic MIDI file, you're not
+      guaranteed to reconstruct the "main" melody.</p>
```

Comment:
I tried out the demo and it's fun! You may want to either let someone upload several midi files, or add a little text to let people know they should have a longer midi file (at least something more than a couple of bars) so that it doesn't just play that one thing over and over.

Alternatively, having an example midi for people to try without finding their own could do the trick.

Reply:
+1

Reply:
You can also add something like "Try training on a single full song to get outputs that sound like variations on it, or train on multiple songs to get samples that combine various characteristics of them."

Reply:
I added both Adam's words, and changed the demo to actually allow uploading multiple files. WDYT?